README.md

```diff
@@ -91,18 +91,25 @@ Open JupyterLab: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybind
 
 ## Development
 
-```shell
-mamba create -c conda-forge -y -n gator python jupyterlab=3
-mamba install -c conda-forge -y -n gator --file requirements_dev.txt
-conda activate gator
```

Comment:
We may want to keep these lines? Setting up a separate mamba/conda env for development should be a good practice here :)

Reply:
Good point, I've added these back in (no need to specify jupyterlab version).